build.sbt

```diff
@@ -10,11 +10,10 @@ lazy val NGHelpers = (project in file("helpers"))
   .settings(
     name := "play-testng-helpers",
     scalaVersion := "2.11.11",
-    crossScalaVersions := Seq("2.11.11", "2.10.6"),
```

Comment:
Please change this to `crossScalaVersions := Seq("2.11.11", "2.12.4"),`, since we need to support both Scala 2.11 and 2.12